src/Tools/dotnet-trace/CommandLine/Commands/CollectCommand.cs

```diff
@@ -222,7 +222,7 @@ private static void PrintProviders(IReadOnlyList<EventPipeProvider> providers, D
             Console.Out.Write(String.Format("{0, -40}","Provider Name"));  // +4 is for the tab
             Console.Out.Write(String.Format("{0, -20}","Keywords"));
             Console.Out.Write(String.Format("{0, -20}","Level"));
-            Console.Out.Write("Enabled By\n");
+            Console.Out.Write("Enabled By\r\n");
```

Comment:
nit: use `Console.Out.WriteLine`. Do we know how is `\r\n` behaving on non-Windows terminal?

Reply:
Yeah, I tested it on my Linux terminals and `\r\n` prints fine.